File: recursion-tutorial-fc/decimalToBinary.js
const decimalToBinary = (number) => {

    if(number == 0) return "";

    // console.log(Math.floor(number / 2));
    return decimalToBinary(Math.floor(number / 2)) + ""  + number % 2;
}

console.log(decimalToBinary(5));

File: recursion-tutorial-fc/DecimalToBinary.java
public class DecimalToBinary {

    public static String convert(int number) {

        if(number == 0) return "";

        return convert(number / 2) + "" + number % 2;
    }

    public static void main(String[] args) {

        System.out.println(DecimalToBinary.convert(2));
    }
}